scripts/libast_prereq.sh

```diff
@@ -25,9 +25,15 @@ fi
 
 cd "$MESON_BUILD_ROOT"
 
+gcc_major_version=$(cc -dumpversion)
+if [ "$gcc_major_version" -ge 9 ]
```

Comment:
This will fail if `cc` is an alias for `clang` as `clang -dumpversion` outputs an string like `4.2.1` (the current value on macOS).

Reply:
In fact, it's kind of weird that it only outputs the major version on gcc.

Reply:
Also, FWIW, if we just got rid of the builtin `getconf` command this awful script could be eliminated. See issue #1118.

Reply:
Fixed.